spec/core_ext/object/descendants_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Class do
+  before do
```

Comment:
you can, but since anonymous classes don't have names, setting them ends up becoming a bit redundant